README.md

```diff
@@ -1,6 +1,6 @@
 # PHP-DI integration with Slim
 
-This package configures Slim 3 to work with the [PHP-DI container](http://php-di.org/).
+This package configures Slim 4 to work with the [PHP-DI container](http://php-di.org/).
```

Comment:
I think you can even remove the version (since older versions will work with Slim 3). It's now becoming "a Slim integration" :)